packages/core/strapi/lib/commands/transfer/transfer.js

```diff
@@ -86,8 +86,8 @@ module.exports = async (opts) => {
   }
 
   const engine = createTransferEngine(source, destination, {
-    versionStrategy: 'ignore', // for an export to file, versionStrategy will always be skipped
-    schemaStrategy: 'ignore', // for an export to file, schemaStrategy will always be skipped
+    versionStrategy: 'exact', // for an export to file, versionStrategy will always be skipped
```

Comment:
If we're targeting exact, you can remove the comments.

Reply:
Done. I'm not 100% sure on strict but I changed it; we initially said it should be exact, but I do think strict is the safest option.